Firebase/Messaging/Public/FIRMessaging.h

```diff
@@ -413,14 +413,18 @@ NS_SWIFT_NAME(Messaging)
 #pragma mark - Topics
 
 /**
- *  Asynchronously subscribes to a topic.
+ *  Asynchronously subscribes to a topic. This uses a FCM Token to identify
```

Comment:
nit: Header comment indent spacing is inconsistent in this file

Reply:
Done.